library/std/src/net/socket_addr.rs

```diff
@@ -133,7 +133,7 @@ impl SocketAddr {
     /// ```
     #[stable(feature = "ip_addr", since = "1.7.0")]
     #[must_use]
-    #[rustc_const_unstable(feature = "const_socketaddr", issue = "82485")]
+    #[rustc_const_stable(feature = "const_socketaddr", since = "1.67.0")]
```

Comment:
We could use `CURRENT_RUSTC_VERSION` so that the version number can be triaged on beta bump, see https://github.com/rust-lang/rust/pull/100591.

Reply:
Sorry for being so slow. Done!